composer.json

```diff
@@ -121,6 +123,7 @@
         "drupal/views_bulk_operations": "^3.6",
         "drupal/xmlsitemap": "^1.0",
         "ext-imagick": "*",
+        "onlyextart/colorbox": "dev-master#e58476becbc89dc671093d1bcd9f99b2167fa8f7",
```

Comment:
Is this needed for the `drupal/colorbox` module above?  Any chance we can use a stable version rather than a commit hash on `master`?

Reply:
+1

Reply:
It doesn't have a Stable version. it's the best packagist version i could find. It's either this, or a zip file with the colorbox library. and then we'd need to have more repositories and composer settings. it seemed like a pain.